applications/API/forms.py

```
@@ -0,0 +1,495 @@
+from django.forms import BooleanField, CharField, ChoiceField, ModelForm, widgets
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+from django_select2 import forms as s2forms
+from loguru import logger
+
+from applications.API.models import Insult, InsultReview
+from common.metrics import metrics
+from common.metrics import metrics
+
+# Module-level caches
+_cached_choices: Union[None, List[Tuple[int, str]]] = None
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)